### PR TITLE
feature: redis observability

### PR DESCRIPTION
**Pull request by ggramal**

### CODE REVIEW

**Review by devops-dmitry — APPROVED** [review]

File: prometheus/values.yaml [hunk 1]
```diff
@@ -431,28 +431,45 @@ prometheus-rules:
...
+          Redis_missing_master:
+            enabled: true
+            for: "5m"
+            description: "Redis cluster has no node marked as master."
+            expr: count by (job) (redis_instance_info{role="master"}) == 0
+          Redis_too_many_masters:
+            enabled: true
+            for: "5m"
+            description: "Redis cluster has too many({{ $value }}) nodes marked as master. "
+            expr: count by (job) (redis_instance_info{role="master"}) > 1
```
[the remark this answers is not in the document]
Comment (ggramal) [reply prometheus/values.yaml:450]: yeah makes sense how can i do it ))

[the remark this answers is not in the document]
Comment (devops-dmitry) [reply prometheus/values.yaml:450]: add code:

[the remark this answers is not in the document]
Comment (ggramal) [reply prometheus/values.yaml:450]: ok let me have a look

[the remark this answers is not in the document]
Comment (ggramal) [reply prometheus/values.yaml:458]: please check

**Review by devops-dmitry — APPROVED** [review]

**Pull request merged**